### PR TITLE
created test suite for group phase

### PR DESCRIPTION
**Pull request by NikAiyer**
Can do the following tests:

verifying that the robots group cards
creating a new group
adding a new reflection during grouping
editing a new reflection during grouping
deleting a reflection after it has been created
renaming a group
advancing to voting

### CODE REVIEW

**Comment by mattkrick** [comment]
i get 1 failing test when run, maybe a race condition?
Screen Shot 2020-03-20 at 5 37 25 PM

**Comment by NikAiyer** [comment]
Yeah the robot groups cards together while the other tests are running. The tests usually finish right around when it's done grouping. But I guess sometimes it'll finish before then. Putting a wait before that test runs would probably allow the robot to complete its task first before I verify the elements. Should I just do that?

File: packages/client/components/GroupingKanbanColumn.tsx [hunk 1]
```diff
@@ -109,15 +110,17 @@ const GroupingKanbanColumn = (props: Props) => {
...
       </ColumnHeader>
-      <ColumnBody isDesktop={isDesktop} {...{[DragAttribute.DROPZONE]: promptId}}>
+      <ColumnBody data-cy={`group-column-${question}-body`} isDesktop={isDesktop} {...{[DragAttribute.DROPZONE]: promptId}}>
         {reflectionGroups
           .filter((group) => {
             // group may be undefined because relay could GC before useMemo in the Kanban recomputes >:-(
             return group && group.reflections.length > 0
           })
           .map((reflectionGroup) => {
+            const groupIndex = reflectionGroups.indexOf(reflectionGroup)
```
[the remark this answers is not in the document]
Comment (NikAiyer) [reply packages/client/components/GroupingKanbanColumn.tsx]: Yeah that makes more sense haha. I'll add that in now.

**Comment by mattkrick** [comment]
another phase in the books! NICE WORK!!!

**Pull request merged**